contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Mozilla Careers",
+    "description": "Lumbergh is the Mozilla careers website. It is a playdoh-based site that displays open jobs.",
+    "repository": {
+        "url": "https://github.com/mozilla/lumbergh",
+        "license": "BSD 3-Clause"
```

Comment:
Talked about this in person, the LICENSE file is BSD 3-Clause because it took the license from Playdoh.